pelican/urlwrappers.py

```diff
@@ -122,3 +122,7 @@ def __init__(self, name, *args, **kwargs):
 
 class Author(URLWrapper):
     pass
+
+
+class Index(URLWrapper):
+    pass
```

Comment:
`URLWrapper._from_settings` is using `self.__class__.__name__.upper()` and key to generate setting name. In this case we need INDEX_SAVE_AS setting value. That is also reason  we have:
```python
class Category(URLWrapper):
    pass


class Author(URLWrapper):
    pass
```